src/main/resources/application.yaml

```diff
@@ -0,0 +1,34 @@
+application:
+  group: zagrodnicy (Bycza-zagroda)
+  version: 0.5.0
+
+spring:
+  banner:
+    location: classpath:banner.txt
+
+  datasource:
+    driver-class-name: com.mysql.cj.jdbc.Driver
+
+  jpa:
+    database-platform: org.hibernate.dialect.MySQL8Dialect
+    open-in-view: false
+    generate-ddl: false
+    hibernate:
+      ddl-auto: validate
+    #do debugowania sql
+    show-sql: true
+    properties:
+      hibernate:
+        format_sql: true
+
+# OpenApi-UI
+springdoc:
+  api-docs:
+    path: /docs/api-docs
+    enabled: true
+  swagger-ui:
+    path: /docs/swagger
+    enabled: true
+
+jwt:
+  secret: Blabla
```

Comment:
Why does the secret have the value "blabla" instead of the part of a very easy to type, remember and in general obvious obviousness such as [ "password" | "secret" | "admin1" | "Ja ro sław Pol skę zbaw" ] ?

Reply:
This value was set during development and I haven't changed it. I think we should use an environment variable to set this value before release. I'm adding a comment so we can remember this.